src/io.js

```diff
@@ -0,0 +1,12 @@
+const fs = require('fs');
+
+const saveImage = (_editionCount, canvas, buildDir) => {
+  fs.writeFileSync(
+    `${buildDir}/images/${_editionCount}.png`,
+    canvas.toBuffer('image/png')
+  );
+};
+
+module.exports = {
```

Comment:
Module for read/write operations. Not all the related methods were moved to this module, it can be done later if we want. Would be good to add specific tests for this module as well.